client/src/components/Donation/PaypalButton.js

```diff
@@ -41,8 +41,8 @@ export class PaypalButton extends Component {
     const { amount, duration } = this.state;
     const { skipAddDonation = false } = this.props;
```

Comment:
```suggestion
    const { isSignedIn = true } = this.props;
```

Reply:
Should we be defaulting to `true`? I do not see it happening anywhere else in the codebase.

Reply:
I just passed the isSignedIn to the modal as well, so thins change is not necessary.